Dockerfile

```diff
@@ -37,6 +37,15 @@ COPY --from=temporal-builder /temporal/temporal-server /usr/local/bin
 FROM temporal-server AS temporal-auto-setup
 CMD /start.sh autosetup
 
+##### Debug configuration for Temporal with additional set of tools #####
+FROM temporal-server as temporal-debug
+# iproute2 contains tc, which can be used for traffic shaping in resiliancy testing. 
+RUN apk add iproute2
+# Add debug configuration file.
+COPY develop/debug/configure.sh /configure.sh
```

Comment:
Let's place it under `docker` dir and rename to `configure-debug.sh`.

Reply:
Sounds good, addressed.